src/lora/lorawan_mac_region.erl

```diff
@@ -82,20 +82,20 @@
 }).
 
 -define(AU915_PAYLOAD_SIZE_MAP, #{
-    0 => 59,
```

Comment:
AU915 and US915 Uplink maximums are similar, just offset by 2.

Reply:
My understanding is that the maximums for the US915 payload (MACPayload - 8) are:
**11, 53, 125, 242,** 242, x, x, x, 53, 129, 242, 242, 242, 242
Whereas the corresponding maximums in AU915 are:
**51, 51, 51, 115,** 242, 242, 242, x, 53, 129, 242, 242, 242, 242 (when the uplink dwell time is off)
And:
x,  x, 11, 53, 125, 242, 242, x, 53, 129, 242, 242, 242, 242 (when the uplink dwell time is on)

I assumed from the code that you weren't handling the uplink dwell time.
It isn't relevant to the maximum downlink sizes, so it's probably best to list the larger values.